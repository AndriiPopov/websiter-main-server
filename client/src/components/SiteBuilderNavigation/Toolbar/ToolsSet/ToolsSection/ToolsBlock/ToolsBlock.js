import React from 'react';

import classes from './ToolsBlock.module.css';

export const toolsBlock = props => {
    return (
        <td className={classes.Block}>
            {props.children}
        </td>
    );
};

export default toolsBlock;