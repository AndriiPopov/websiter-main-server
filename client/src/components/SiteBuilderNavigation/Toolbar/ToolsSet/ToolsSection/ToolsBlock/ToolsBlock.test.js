import React from 'react';
import {shallow, configure} from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import { toolsBlock as ToolsBlock } from './ToolsBlock';

configure({adapter: new Adapter()});

describe('ToolsBlock', () => {
    it('should be defined', () => {
        expect(ToolsBlock).toBeDefined();
    });

    it('it should render a td with children', () => {
        const tree = shallow(
            <ToolsBlock>
                <p>Test</p>
            </ToolsBlock>
        );
        expect(toJson(tree)).toMatchSnapshot();
    });
});