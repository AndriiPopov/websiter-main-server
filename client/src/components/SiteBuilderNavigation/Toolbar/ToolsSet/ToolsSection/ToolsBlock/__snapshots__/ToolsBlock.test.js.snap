// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolsBlock it should render a td with children 1`] = `
<td
  className="Block"
>
  <p>
    Test
  </p>
</td>
`;
